README.md

```diff
@@ -99,3 +99,62 @@ So as to specify right the target for scheme to run:
 The last step is to remove the red named folder `Removable Resources`.
 
 <img src="./resources/images/readme/scheme_remove-folder.png" width="500">
+
+</br>
+</br>
+</br>
+
+# Tuist Installation and Documentations
+
+</br>
+
+Run the following command in your terminal for the Tuist installation:
+
+</br>
+
+```bash
+bash <(curl -Ls https://install.tuist.io)
+```
+</br>
+
+Documentation : [Tuist Official Documents](https://docs.tuist.io/tutorial/get-started)
+
+</br>
+
+
+## How to use
+
+</br>
+
+- Copy the whole directory `Tuist` and its content to one directory up of your project's directory.
+
+    *Assume your future project in `Projects` directory*
+
+    ```
+    Projects
+    |-- Tuist
+        |-- Templates
+            |-- mvvm
+                |-- AppDelegate.swift
+                |-- mvvm.swift
+                |-- Project.stencil
+    |-- <Your future project's directory>
+    └── ...
+    ```
+</br>
+
+- Navigate to your future project's directory and run
+
+    ```bash
+    tuist init -t mvvm --name <Name of project>    
```

Comment:
I would prefer something like this `tuist init -t nimble-template --name <name of project>`. The project not only contains `MVVM` (MVVM is just an architecture pattern in the `Presentation Layer`).

Reply:
Please re-check template folder too since the current commit has folder name `mvm`